Formula/libxspf.rb

```diff
@@ -1,14 +1,9 @@
 class Libxspf < Formula
   desc "C++ library for XSPF playlist reading and writing"
   homepage "https://libspiff.sourceforge.io/"
-  url "https://downloads.xiph.org/releases/xspf/libxspf-1.2.0.tar.bz2"
-  mirror "https://ftp.osuosl.org/pub/xiph/releases/xspf/libxspf-1.2.0.tar.bz2"
-  sha256 "ba9e93a0066469b074b4022b480004651ad3aa5b4313187fd407d833f79b43a5"
-
-  livecheck do
```

Comment:
Is it possible to add a new livecheck?

Reply:
No longer needs a custom one, since it can now use the built-in Sourceforge strategy.
```
$ brew livecheck libxspf -d
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/libxspf.rb

Formula:          libxspf
Livecheckable?:   No

URL:              https://downloads.sourceforge.net/project/libspiff/Sources/1.2.1/libxspf-1.2.1.tar.bz2
Strategy:         Sourceforge
URL (strategy):   https://sourceforge.net/projects/libspiff/rss
Regex (strategy): /url=.*?\/libspiff\/files\/.*?[-_\/](\d+(?:[-.]\d+)+)[-_\/%.]/i

Matched Versions:
1.2.1, 1.2.0, 1.1.0, 1.0.0, 1.0, 0.8.4, 0.8.3, 0.8.2, 0.8.1, 0.8.0, 0.7.3, 0.7.2, 0.7.1, 0.7.0, 0.6.5, 0.6.4, 0.6.3
libxspf : 1.2.1 ==> 1.2.1
```

Reply:
Ok, thanks. I did not know we had this. Nice :)